tests/middlewareTypes.test.tsx

```diff
@@ -262,15 +262,17 @@ describe('counter state spec (double middleware)', () => {
   it('devtools & immer', () => {
     __DEV__ = false
     const useBoundStore = create<CounterState>()(
-      devtools(
-        immer((set, get) => ({
-          count: 0,
-          inc: () =>
-            set((state) => {
-              state.count = get().count + 1
-            }),
-        })),
-        { name: 'prefix' }
+      immer(
+        devtools(
+          (set, get) => ({
+            count: 0,
+            inc: () =>
+              set((state) => {
```

Comment:
Let's add an action with payload to this `set` call (like you did previously) as a test for types.